theatre/studio/src/propEditors/simpleEditors/RgbaPropEditor.tsx

```diff
@@ -1,5 +1,6 @@
 import type {PropTypeConfig_Rgba} from '@theatre/core/propTypes'
-import type {Rgba} from '@theatre/shared/utils/color'
+import type {Rgba} from '@theatre/shared/utils/color';
+import { validHexRegExp} from '@theatre/shared/utils/color'
```

Comment:
I'm surprised the commit hook isn't picking up these style inconsistencies. Or did you run `git commit --no-verify`? ;)

Reply:
Nope, I simply ran `git commit`. Interesting indeed. I'll fix this one (and will check if there's more).